manifests/s/Saladict/app/3.0.6/Saladict.app.locale.en-US.yaml

```
@@ -0,0 +1,31 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Saladict.app
+PackageVersion: 3.0.6
+PackageLocale: en-US
+Publisher: allentown521
+PublisherUrl: https://github.com/allentown521/
+PublisherSupportUrl: https://github.com/allentown521/
+PrivacyUrl: https://github.com/allentown521/
+Author: allentown521
+PackageName: app
+PackageUrl: https://github.com/allentown521/saladict
+License: GPL-3.0
+LicenseUrl: https://github.com/allentown521/saladict/blob/master/LICENSE
+Copyright: Copyright (c) Focus Apps
+CopyrightUrl: https://github.com/allentown521/saladict/blob/master/LICENSE
+ShortDescription: 一个跨平台的划词翻译和OCR软件 | A cross-platform software for text translation and recognize.
+Description: 一个跨平台的划词翻译和OCR软件 | A cross-platform software for text translation and recognize.
+Moniker: translate
+Tags:
+- translation
+- tauri
+- saladict
+- translate
+ReleaseNotes: 3.0.6
```

```suggestion
ReleaseNotes: |
  ### New feature:
    - add builtin lingva service
    - add support for hebrew
    - improve ui translation
  ### Bugs fixed:
    - builtin ecdict can not be save
```

You've already given the release notes, so don't just use the version number.
你在 CHANGELOG 中已经给过发行说明了，这里就别再用版本号了。